src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -3024,6 +3025,12 @@ export default defineComponent({
     const invertValueChangeContentOrder = ref(false)
     let valueChangeTimeout = null
 
+    function showOverlayControls() {
+      if (container.value) {
+        container.value.dispatchEvent(new MouseEvent('mousemove'))
+      }
```

Comment:
Fake event is not needed, method available to make UI shown
https://shaka-player-demo.appspot.com/docs/api/shaka.ui.Controls.html
```suggestion
      ui.getControls().showUI()
```